geode-apis-compatible-with-redis/src/main/java/org/apache/geode/redis/internal/netty/Coder.java

```diff
@@ -155,12 +156,12 @@ private static void writeCollectionOrString(ByteBuf buffer, Object next) throws
   public static ByteBuf getScanResponse(ByteBuf buffer, BigInteger cursor,
       List<?> scanResult) {
     buffer.writeByte(ARRAY_ID);
-    buffer.writeBytes(intToBytes(2));
+    buffer.writeByte('2');
```

Comment:
Per a comment at the top of `StringBytesGlossary` (which was originally in this class before being pulled out):
```
 * Important note
 * <p>
 * Do not use '' <-- java primitive chars. Redis uses {@link Coder#CHARSET} encoding so we should
 * not risk java handling char to byte conversions, rather just hard code {@link Coder#CHARSET}
 * chars as bytes
 ```
Do we still want to follow this? If so, I think the solution is to introduce a `byte` constant to the `StringBytesGlossary` class for this char. We already have constants for 0 and 1, so this wouldn't be too out of the ordinary.

Reply:
Thanks for pointing this out. It was helpful and I think it is a good idea. I have switch all the code over to using these byte constants instead of java char literals